modules/benchmark/R/load_data.R

```diff
@@ -26,13 +27,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    #will be made into R package
+    install_github("yan130/bd.r")
+    
+    key   = get_key("https://bd-api.ncsa.illinois.edu",username,password) #generic
+    token = get_token("https://bd-api.ncsa.illinois.edu",key)
+    #output_path = "/fs/data3/jam2767/veg_test" where are we putting converted file?
+    convert_file(url = "https://bd-api.ncsa.illinois.edu", 
```

Comment:
you aren't assigning the output of convert_file to anything. You need to do that to get the new filename. Also, you need to add a check after this make sure that convert_file returned something. If it didn't you need to return an informative error message (make user to use logger not print).